sdk/eventhub/eventhubs-checkpointstore-tables/src/index.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/// <reference lib="esnext.asynciterable" />
```

Comment:
Is this reference needed?

Reply:
it was in the equivalent file for checkpoint-blob too.

Reply:
Sure, but do we need it in this package? If you remove it, does the package still compile?

This should only be required if we are using async iterables, which I don't think we are at the moment.